vllm/entrypoints/openai/protocol.py

```diff
@@ -183,6 +183,7 @@ class CompletionRequest(BaseModel):
     guided_json: Optional[Union[str, dict, BaseModel]] = None
     guided_regex: Optional[str] = None
     guided_choice: Optional[List[str]] = None
+    truncate_prompt_tokens: Optional[int] = None
```

Comment:
nit can you make this a constrained integer: https://docs.pydantic.dev/2.3/api/types/#pydantic.types.conint

Reply:
done